packages/clerk-js/src/ui/components/OrganizationSwitcher/OtherOrganizationActions.tsx

```diff
@@ -86,7 +109,133 @@ export const OrganizationActionList = (props: OrganizationActionListProps) => {
           </PreviewButton>
         ))}
       </Box>
+
+      {(userInvitations.count ?? 0) > 0 && (
+        <Flex
+          direction='col'
+          elementDescriptor={descriptors.organizationSwitcherPopoverInvitationActions}
+        >
+          <Text
+            variant='smallRegular'
+            sx={t => ({
+              minHeight: 'unset',
+              height: t.space.$12,
+              padding: `${t.space.$3} ${t.space.$6}`,
+              display: 'flex',
+              alignItems: 'center',
+            })}
+            // Handle plurals
+            localizationKey={localizationKeys(
+              (userInvitations.count ?? 0) > 1
+                ? 'organizationSwitcher.invitationCountLabel_many'
+                : 'organizationSwitcher.invitationCountLabel_single',
+              {
+                count: userInvitations.count,
+              },
+            )}
+          />
+          <Box
```

Comment:
🔧 We should not render an empty box if no userInvitations are available

Reply:
`{(userInvitations.count ?? 0) > 0 && (`
The above  code prevents that, but it is placed on the outer containers